etc/profile-m-z/youtube-dl-gui.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for youtube-dl-gui
+# Description: A cross platform front-end GUI of the popular youtube-dl media downloader
+include youtube-dl-gui.local
+# This file is overwritten after every install/update
+include globals.local
+
+#These are blacklisted by disable-interpreters.inc
+include allow-python2.inc
+include allow-python3.inc
+
+noblacklist ${HOME}/.config/youtube-dlg
+noblacklist ${VIDEOS}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/youtube-dlg
+whitelist ${HOME}/.config/youtube-dlg
+whitelist ${VIDEOS}
```

Comment:
Just thinking: `whitelist ${DOWNLOADS}`, yes / no? IDK

Reply:
let's go for `whitelist ${DOWNLOADS}` instead of `whitelist ${VIDEOS}`